app/src/main/java/ch/sdp/vibester/helper/Helper.kt

```diff
@@ -10,7 +10,7 @@ import kotlinx.coroutines.async
 import kotlinx.coroutines.launch
 import java.util.concurrent.TimeUnit
 
-class ImageHelper {
+class Helper {
```

Comment:
if this class deals only with unage, maybe it is better to leave it as ImageHelper?